cmd/object-handlers.go

```diff
@@ -3033,14 +3036,14 @@ func (api objectAPIHandlers) PutObjectTaggingHandler(w http.ResponseWriter, r *h
 		return
 	}
 
-	tagging, err := tagging.ParseTagging(io.LimitReader(r.Body, r.ContentLength))
+	tags, err := tags.UnmarshalXML(io.LimitReader(r.Body, r.ContentLength), true)
```

Comment:
UnmarshalXML is usually used for xml.UnmarshalXML why isn't tags.ParseXML a better option here?